packages/core/src/sass/app/_common/components/onfido-container.scss

```diff
@@ -72,191 +72,688 @@
+                grid-gap: 1rem;
+            }
+
+            align-items: center;
+
+            @include mobile {
+                grid-template-areas: 'section-side-note' 'section-content';
+            }
+
+            &-side-note {
+                grid-area: section-side-note;
+
+                .account-poa__upload-box-dashboard {
+                    @include mobile {
+                        align-items: flex-start;
+                    }
+                }
+
+                @include mobile {
+                    margin-top: unset;
+                    width: 100%;
+                    margin-bottom: 0.8rem;
+                }
+            }
+
+            &-content {
+                grid-area: section-content;
+
+                @include mobile {
+                    width: 100%;
+                }
             }
         }
     }
+}
+
+.proof-of-identity {
+    width: 100%;
+    height: 100%;
+
+    &__main-container {
+        max-width: 68.2rem;
+    }
 
-    &__section {
-        display: grid;
+    .min-height {
+        min-height: 50vh;
 
-        @include desktop() {
-            grid-template-areas: 'section-side-note section-content';
-            grid-template-columns: auto 1fr;
-            grid-gap: 1rem;
+        @include mobile {
+            min-height: unset;
         }
+    }
 
+    &__container {
+        display: flex;
+        flex-direction: column;
+        justify-content: flex-start;
         align-items: center;
 
         @include mobile {
-            grid-template-areas: 'section-side-note' 'section-content';
+            min-height: unset;
+            overflow-y: scroll;
+            justify-content: center;
         }
 
-        &-side-note {
-            grid-area: section-side-note;
+        &--reset {
+            align-items: normal;
+
+            @include mobile {
+                overflow-y: unset;
+                padding: 1.6rem;
+            }
 
-            .account-poa__upload-box-dashboard {
+            .proof-of-identity__submit-button {
                 @include mobile {
-                    align-items: flex-start;
+                    margin: unset;
+                    width: 100%;
                 }
             }
 
-            @include mobile {
-                margin-top: unset;
-                width: 100%;
-                margin-bottom: 0.8rem;
+            .account-form__footer {
+                @include mobile {
+                    padding: 1.6rem 2.4rem;
+                }
+            }
+
+            .form-body {
+                @include desktop {
+                    z-index: 5;
+                }
             }
         }
 
-        &-content {
-            grid-area: section-content;
+        .icon {
+            width: 128px;
+            height: 128px;
+            margin-top: 5.2rem;
+            margin-bottom: 2.6rem;
 
             @include mobile {
-                width: 100%;
+                width: 72px;
+                height: 72px;
```

Comment:
rem here too

Reply:
This is for icon, so I think it shouldn't be relative